src/app/grants/mainGrant.tsx

```diff
@@ -15,11 +14,41 @@ export default function MainGrant({
   return (
     <div className="flex flex-col">
       <GrantsHero />
-      <Eligibility />
-      <StartGrant setPage={setPage} />
-      <Perks />
-      <Cohort setPage={setPage} />
-      <CohortStructure />
+      <BtrustGrant />
+      {/* <Eligibility /> */}
+      <AllGrants setPage={setPage} />
+      <FaqContainer
+        faqs={[
+          {
+            answer:
+              "Btrust Grants provide financial support to developers, educators, and event organizers working to strengthen the Bitcoin open-source ecosystem. We offer grants for developer contributions, education programs, research, and Bitcoin-only events to remove financial barriers and enable impactful contributions.",
```

Comment:
Cleaned to take out things like the "research" and clarify the dev side:

```suggestion
              "Btrust Grants provide financial support to developers, educators, and event organizers working to strengthen the Bitcoin open-source ecosystem. We offer grants for Bitcoin Open-Source development, Bitcoin education programs, and Bitcoin-only events to remove financial barriers and enable impactful contributions.",
```